Index: closure/goog/net/browserchannel.js
===================================================================
--- closure/goog/net/browserchannel.js	(revision 1850)
+++ closure/goog/net/browserchannel.js	(working copy)
@@ -2245,7 +2245,9 @@
   if (hostPrefix) {
     throw new Error('Can\'t create secondary domain capable XhrIo object.');
   } else {
-    return new goog.net.XhrIo();
+    var xhr = new goog.net.XhrIo();
+    xhr.setWithCredentials(true);
+    return xhr;
   }
 };
 
